src/bundle/bundle.ts

```diff
@@ -12,7 +26,9 @@ export const bundle = async (options: BundleOptions) => {
     bundle: true,
     format: "esm",
     write: false,
+    platform: "node",
```

Comment:
ok so this is a bit of a mess currently but this flag is set by `src/cli/commands/bundle.ts`